.gitignore

```diff
@@ -5,7 +5,6 @@ reports/
 dist/
 tests/phpunit.xml
 vendor/
-box.phar
```

Comment:
If you commit it in the repo, you should at least ignore it from the archive (using `.gitattributes`) so that users of the package don't get the box phar downloaded in their projects.

Reply:
This was already reverted, see #694 + #700.